src/conn/conn_cache_pool.c

```diff
@@ -154,7 +154,7 @@ __wt_cache_pool_config(WT_SESSION_IMPL *session, const char **cfg)
 	 *  - We are joining a cache pool for the first time (including
 	 *  creating the pool) - use the chunk size; that's the default.
 	 */
-	if (__wt_config_gets(session, &cfg[1],
+	if (__wt_config_gets(session, cfg,
```

Comment:
Maybe I'm missing something, but why would this code exclude the base configuration?

Note 25 lines before, in the `!created` code, it's using `cfg`, not `&cfg[1]` -- why is this code, which is executed in that case, using `&cfg[1]`?